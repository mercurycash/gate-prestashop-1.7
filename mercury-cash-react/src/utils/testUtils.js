import { render } from "@testing-library/react";
import { Provider } from "outstated";
import { store } from "../state/state";

export const customRender = (ui) => {
  return render(
    <Provider stores={ [ store ] }>{ ui }</Provider>
  )
}
