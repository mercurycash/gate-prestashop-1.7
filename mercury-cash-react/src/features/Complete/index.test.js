import { Complete } from "./index";
import { customRender } from "../../utils/testUtils";

describe('<Complete/>', () => {
  const spy = jest.spyOn(global.console, "error")

  it('Render Complete Component', async () => {
    const { container } = customRender(<Complete/>)
    expect(container).toMatchSnapshot()
    expect(spy).not.toHaveBeenCalled()
  })
})