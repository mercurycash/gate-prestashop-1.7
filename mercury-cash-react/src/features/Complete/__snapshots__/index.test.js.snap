// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Complete/> Render Complete Component 1`] = `
<div>
  <div
    class="mercury-cash__complete"
  >
    <h3
      class="mercury-cash__title mercury-cash__title--big"
    >
      completed
    </h3>
    <img
      alt=""
      src="done.png"
    />
    <h3
      class="mercury-cash__title mercury-cash__title--big"
    >
      thankYou
       
      <span>
        forPaying
      </span>
    </h3>
    <svg>
      logo-with-text.svg
    </svg>
  </div>
</div>
`;
